scripts/natural_language_inference/nlidataset.py

```diff
@@ -0,0 +1,69 @@
+"""
+nlidataset.py
+
+Part of NLI script in gluon-nlp.
+Copyright 2018 Mengxiao Lin <linmx0130@gmail.com>
+"""
+
+import json
+import mxnet.gluon.data as gdata
+
+class NLIDataItem:
```

Comment:
In gluonnlp, we intentionally avoid using an item class so that it's more flexible. You may use a simple tuple to achieve the same and put the transformation logic in transform functions.